.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-  "typescript.tsdk": "node_modules/typescript/lib"
-}
\ No newline at end of file
+  "typescript.tsdk": "node_modules/typescript/lib",
```

Comment:
Can you try reverting this change? It's extremely weird if Prettier was responsible for it. Prettier shouldn't touch `.vscode` directory. And trailing commas isn't really valid json syntax.

Reply:
That's done